e2e-tests/gef/cypress/e2e/about-exporter.spec.js

```diff
@@ -39,7 +39,7 @@ context('About Exporter Page', () => {
   });
 
   beforeEach(() => {
-    Cypress.Cookies.preserveOnce('connect.sid');
+    Cypress.Cookies.preserveOnce('dtfs-session');
```

Comment:
`Cypress.Cookies.preserveOnce('');` is deprecated in the latest version. If we need to update this line, then we may as well upgrade Cypress to its latest version and update the way we handle sessions in e2e tests --> https://docs.cypress.io/guides/references/changelog#12-0-0